content/getting-started/install.md

```diff
@@ -22,44 +22,80 @@ Pick the most suitable instructions for your operating system:
 
 ### macOS
 
-On a Mac you can use brew:
+On a Mac you can use [brew](https://brew.sh/):
 
 ```shell
 brew tap jenkins-x/jx
-brew install jx 
+brew install jx
 ```
 
-Or if you have not installed [brew](https://brew.sh/) and prefer to install by hand:
+To install Jenkins X on macOS without brew, download the `.tar` file, and unarchive it in a directory where you can run the `jx` command.
+
+1.  Download the `jx` binary archive using `curl` (where
+    \[LATEST\_VERSION\] is the most current version of Jenkins X
+    released, which is 2.0.46) and pipe (`|`) the compressed archive to
```

Comment:
That's not the current version - let's remove this hardcoded version number and add a link to https://github.com/jenkins-x/jx/releases/ so that user can check what's the current version :)